syntax = "proto3";

package benchmark_test;

message LoginMessage {
  int32 id = 1;
  string password = 2;
}

message LoginResult{
  int32 code = 1;
  string error = 2;
}
